src/components/Proposer/Proposer.svelte

```diff
@@ -47,52 +63,39 @@
 					{firstname}
 					{lastname}
 				</a>
-				{#if matchedAssemblyRole}
-					{@const { assembly } = matchedAssemblyRole}
-					<!-- TODO: cabinet is not released yet -->
-					{@const isCabinet = assembly.name === AssemblyName.Cabinet}
-					<svelte:element
-						this={isCabinet ? 'p' : 'a'}
-						href={isCabinet ? undefined : `/assemblies/${assembly.id}`}
-						class="text-sm text-black {isCabinet ? '' : 'underline'}"
-					>
+				{#if assembly}
+					<a href={`/assemblies/${assembly.id}`} class="text-sm text-black underline">
```

Comment:
In Svelte you can do
```svelte
href="/assemblies/{assembly.id}"
```
which is more concise

Reply:
Done. Thanks!